internal/server/iigointernal/utilities.go

```diff
@@ -44,3 +44,11 @@ func WithdrawFromCommonPool(value shared.Resources, gameState *gamestate.GameSta
 func depositIntoCommonPool(value shared.Resources, state *gamestate.GameState) {
 	state.CommonPool += value
 }
+
```

Comment:
Any suggestions on a less ugly function would be appreciated! I couldn't think of a way around this...